proto/meta.proto

```diff
@@ -307,3 +308,15 @@ message MembersResponse {
 service MetaMemberService {
   rpc Members(MembersRequest) returns (MembersResponse);
 }
+
+message SystemParams {
+  uint32 barrier_interval_ms = 1;
+  uint64 checkpoint_frequency = 2;
+  uint32 sstable_size_mb = 3;
+  uint32 block_size_kb = 4;
+  double bloom_false_positive = 5;
+  string state_store = 6;
+  string data_directory = 7;
+  string backup_storage_url = 8;
+  string backup_storage_directory = 9;
+}
```

Comment:
Do we need to define every field in proto, since they are just persisted as string in meta 🥵

Reply:
Well, they must be ser/de *somewhere*, so maybe in proto isn't a bad idea.. 🤔

Reply:
It is needed when worker nodes pull system parameters with `MetaClient::register_new`.